src/QMCWaveFunctions/LCAO/SoaLocalizedBasisSet.h

```diff
@@ -211,6 +216,7 @@ class SoaLocalizedBasisSet : public SoaBasisSetBase<ORBT>
 private:
   /// multi walker shared memory buffer
   struct SoaLocalizedBSetMultiWalkerMem;
+  void initializeSpeciesOffsets();
```

Comment:
Please move this function to the end of the class and also document what exactly this function does.
It is not good to mix functions and variables.